.github/workflows/e2e.yml

```
@@ -61,34 +69,23 @@ jobs:
           NODE_ENV: "production"
         working-directory: packages/wrangler
 
-      - name: Find Wrangler
+      - name: Find pnpm global dir
```

No need for abbreviation here

```suggestion
      - name: Find pnpm global directory
```